src/key_pair.rs

```diff
@@ -143,20 +144,22 @@ impl KeyPair {
 		})
 	}
 
-	pub(crate) fn from_raw(
-		pkcs8: &[u8],
+	pub(crate) fn from_raw<'b>(
+		pkcs8: impl Into<Cow<'b, [u8]>>,
```

Comment:
How is this helpful? IMO this kind of trick is usually good for public APIs, but for a private API I'd probably just take `&[u8]` -- it doesn't look like anything is making good use of the possibility of `Cow::Owned` anyway.